requirements/constraints.txt

```diff
@@ -0,0 +1,34 @@
+git+https://github.com/edx/django-pyfs.git@1.0.6#egg=django-pyfs==1.0.6
+git+https://github.com/edx/django-rest-framework.git@1ceda7c086fddffd1c440cc86856441bbf0bd9cb#egg=djangorestframework==3.6.3
+git+https://github.com/edx/edx-lint.git@1.3.0#egg=edx_lint==1.3.0
+git+https://github.com/edx/XBlock.git@xblock-1.0.1#egg=XBlock==1.0.1
+git+https://github.com/edx/xblock-sdk.git@331e50bed2a3c9014d998cc4428e92bf658baf8d#egg=xblock-sdk==0.1.6
+
+bleach==2.1.4
```

Comment:
This is a lot of things to constrain.  Did you have to add all of these 1-by-1?

Reply:
+1

Reply:
Loosened up the constraints slightly to use `>=` for most dependencies but didn't want to mess with them too much... yet. Some still have max versions pinned (for Python 2.7 compatibility?) which I didn't touch.